draft-ietf-tls-ticketrequests.md

```diff
@@ -153,6 +153,15 @@ IANA is requested to Create an entry, ticket_request(TBD), in the existing regis
 for ExtensionType (defined in {{RFC8446}}), with "TLS 1.3" column values being set to
 "CH", and "Recommended" column being set to "Yes".
 
+# Performance Considerations
+
+Servers can send the requested tickets in NewSessionTicket messages at any time after the
+client Finished message. If the server chooses to send a large number of tickets to the client,
```

Comment:
```suggestion
server Finished message (see {{RFC8446}}; Section 4.6.1). A server which chooses to send a large number of tickets to a client
```

Reply:
That section say client finished though... 

`At any time after the server has received the client Finished message, it MAY send a NewSessionTicket message.`

Reply:
It also says this:

```
Note: Although the resumption master secret depends on the client's
   second flight, a server which does not request client authentication
   MAY compute the remainder of the transcript independently and then
   send a NewSessionTicket immediately upon sending its Finished rather
   than waiting for the client Finished.
```

Reply:
Ahhh I see